package.json

```diff
@@ -70,7 +70,7 @@
     "unpackage:vsix": "gulp vsix:release:unpackage",
     "updatePackageDependencies": "gulp updatePackageDependencies",
     "l10nDevGenerateXlf": "npx @vscode/l10n-dev generate-xlf ./package.nls.json ./l10n/bundle.l10n.json --outFile ./loc/vscode-csharp.xlf",
-    "l10nDevImportXlf": "npx @vscode/l10n-dev import-xlf ./loc/vscode-csharp.*.xlf --outDir ./l10n"
+    "l10nDevImportXlf": "npx @vscode/l10n-dev import-xlf ./loc/vscode-csharp.*.xlf --outDir ./l10n && move /l10n/package.nls.*.json ."
```

Comment:
'move' can't be used on bash. (mv should be there)
But for package.nls.json, if a contributor makes a change in package.json and creates a reference to it. That's everything. 
This task should not be run on dev's machine. It should only be run on CI (windows environment, because localization tasks can only be run there)

Reply:
I can't find anything about @vscode/l10n-dev import-xlf to support multiple output-dir. Seems like it can only have one out dir
<img width="662" alt="image" src="https://github.com/dotnet/vscode-csharp/assets/24360909/a513166a-19c1-49eb-b301-77656bed6591">

Reply:
CI is currently linux based I believe

Reply:
Can we just specify '.' as the output dir so we don't have to move it?

Reply:
No, localization CI is windows see https://github.com/dotnet/vscode-csharp/blob/aab28042db1b880823ddd85b3b0fea3f8632df3e/azure-pipelines/loc.yml#L20

Reply:
> Can we just specify '.' as the output dir so we don't have to move it?

Yeah but in that case we need to move `bundle.l10.*.json` to `l10` folder

Reply:
Or it's can be something like this, we put all `bundle.l10.*.json`  and `package.nls.*.json` to root. And change this https://github.com/dotnet/vscode-csharp/blob/aab28042db1b880823ddd85b3b0fea3f8632df3e/package.json#L46

I just feel it's too many things under the root

Reply:
> Yeah but in that case we need to move bundle.l10.*.json to l10 folder

Oooh ok yeah can leave it there and move it

> OneLocBuild Task is required to run on windows

Ah this runs in the separate pipeline - my bad.